compiler/src/iree/compiler/Codegen/ExternalInterfaces/Utils.h

```diff
@@ -74,9 +74,22 @@ struct EncodingLayoutMaterializerAttrExternalModelBase
           if (IREE::Codegen::isIdentityLayout(encodingInfo)) {
             return type.dropEncoding();
           }
+          // For shape inference, mark scalable tiles as dynamic sizes for the
+          // shape inference. Note, scalable tiles that are represented with
+          // static inner tile sizes.
```

Comment:
nit: remove dup words. (please format the comment. I can't format it easily because I edited it on web UI.)

```suggestion
          // Mark scalable tiles as dynamic sizes for the
          // shape inference. Note, scalable tiles that are represented with
          // static inner tile sizes.
```

Reply:
done